hackclub.com.yaml

```
@@ -436,8 +436,18 @@ archived.guide:
   value: hackclub-guide.netlify.com.
 arsenic:
   - ttl: 600
-    type: CNAME
+  - type: CNAME
```

remove this hyphen to resolve the tests

```suggestion
    type: CNAME
```